rllib/policy/torch_policy_v2.py

```diff
@@ -1130,7 +1178,8 @@ def _worker(shard_idx, model, sample_batch, device):
 
                     # Call Model's custom-loss with Policy loss outputs and
                     # train_batch.
-                    loss_out = model.custom_loss(loss_out, sample_batch)
+                    if hasattr(model, "custom_loss"):
```

Comment:
so the reason I added those IsOverridden calls is that checking whether an API exists may not be safe if say model has a default Implementation of custom_loss() that simply raise NotImplementedError.
I don't know if eventually that will be the case for model, just keep this in mind.

Reply:
oh i see. That makes sense.